store/postgres/src/mapping.rs

```diff
@@ -238,8 +236,20 @@ impl Mapping {
         Ok(mapping)
     }
 
+    /// Generate the DDL for the entire mapping, i.e., all `create table`
+    /// and `create index` etc. statements needed in the database schema
     pub fn as_ddl(&self) -> Result<String, fmt::Error> {
-        (self as &dyn AsDdl).as_ddl(self)
+        let mut out = String::new();
+
+        // We sort tables here solely because the unit tests rely on
+        // 'create table' statements appearing in a fixed order
+        let mut tables = self.tables.values().collect::<Vec<_>>();
+        tables.sort_by(|a, b| (&a.position).partial_cmp(&b.position).unwrap());
```

Comment:
[sort_by_key](https://doc.rust-lang.org/std/primitive.slice.html#method.sort_by_key) will be nicer here.

Reply:
It is indeed! Commit 5d7a2d96